### DISCUSSION TITLE
How to make posts in the added collection visible in category tab?

### DISCUSSION THREAD

**Discussion by arcstone09** [academicpages/academicpages.github.io#2008] [Q&A]
I want to make my posts displayed by its category when ‘category’ tab is clicked.

In order to do this, I uncommented the ‘jekyll-archives’ part in _config.yml.

For posts in collections which are already in the template(ex. Blog posts), it works.

However, posts that are located in collection that I added don’t appear when I click the category tab. What should I do?

(To add collection, I changed _config.yml collections part and navigation.yml file, added [collection_name].html file in _pages folder, and wrote new posts with category in the folder named _study that I made. Posts in _study folder is visible in the ‘study’ tab, but not visible in the ‘category’ tab.)

**Comment by rjzupkoii** [comment]
This is related to issue #1655 which is in the pipeline to be addressed.
